account-management-integration-tests/src/test/java/uk/gov/di/accountmanagement/api/UpdateEmailIntegrationTest.java

```diff
@@ -46,6 +48,11 @@ void setup() {
     void shouldCallUpdateEmailEndpointAndReturn204WhenUpdatingEmailIsSuccessful() {
         var internalCommonSubId = setupUserAndRetrieveInternalCommonSubId();
         var otp = redis.generateAndSaveEmailCode(NEW_EMAIL_ADDRESS, 300);
+
+        Map<String, Object> requestParams =
+                new HashMap<>(Map.of("principalId", internalCommonSubId));
+        requestParams.put("clientId", CLIENT_ID);
```

Comment:
You can do:
`requestParams = Map.of("principalId", internalCommonSubId, "clientId", CLIENT_ID)